tests/tests_regenerate_defaults.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Test we can write global config with default configuration
```

Comment:
Could this be improved by setting some option to non-default value, then regenerating the file, then checking if the value is default?

Reply:
sure. I can add also different config and make sure it is propagated too.

Reply:
Done in the 32fd903.

Reply:
Clarified off-PR. I was more thinking about "resetting the config to default" kind of use-case here.

Reply:
Sorry for misunderstanding. Does this reflect what was on your mind better: 4a92434

Reply:
Yes, thanks!